docs/user-manual/en/address-model.md

```diff
@@ -964,3 +965,13 @@ queues created on the matching address. Defaults to 0. Read more about
 `enable-metrics` determines whether or not metrics will be published to any
 configured metrics plugin for the matching address. Default is `true`. Read more
 about [metrics](metrics.md).
+
+`enable-ingress-timestamp` determines whether or not the broker will add its
+timestamp to messages sent to the matching address. When `true` the exact behavior
+will depend on the specific protocol in use. For AMQP messages the broker will
+add a `long` *message annotation* named `x-opt-ingress-timestamp-millis`. For core
+messages (used by the core and OpenWire protocols) the broker will add a property
```

Comment:
Perhaps "`long` property" instead of just "property" ?

Reply:
Will do.